app/Http/Controllers/API/v1/Category/CategoryController.php

```diff
@@ -141,7 +141,8 @@ public function listOfCategories(Request $request)
     {
         $query = request()->query();
 
-        $categories = Category::withCount('subcategories');
+        $categories = Category::withCount('subcategories')
```

Comment:
is the withcount necessary? please remove if not needed. you can refer to this post 

https://stackoverflow.com/questions/35301846/filtering-get-results-in-laravel